api-admin/src/main/resources/api/openapi.yml

```diff
@@ -836,7 +855,7 @@ paths:
           content: { application/json: { schema: { type: object } } }
         '500':
           $ref: '#/components/responses/ServerError'
-  /api/portals/v1/{portalShortcode}/env/{envName}/triggers/{configId}/test:
+  /api/portals/v1/{portalShortcode}/studies/{studyShortcode}/env/{envName}/triggers/{configId}/test:
```

Comment:
Was this broken? I am confused because it seems to be working on development and also in this PR, but there is no corresponding change in `api.tsx` so I feel like I'm missing something.

Reply:
hmm... I'm not sure -- maybe this is a merge artifact.  I'll double-check

Reply:
yup, just did a merge and it cleaned up, thanks for calling that out